itest/assertions.go

```diff
@@ -68,7 +68,7 @@ func assetAnchorCheck(txid, blockHash chainhash.Hash) assetCheck {
 				txid[:])
 		}
 
-		if !bytes.Equal(a.ChainAnchor.AnchorBlockHash, blockHash[:]) {
+		if a.ChainAnchor.AnchorBlockHash != blockHash.String() {
```

Comment:
This commit removes the `bips` sub-directory, which I don't think is intentional.